### DISCUSSION TITLE
Why just #StandWithUkraine and not others?

### DISCUSSION THREAD

**Discussion by salman-ibrahim** [composer/composer#11098] [General]
It's annoying or should I say disturbing that i have to see #StandWithUkraine every single time I run something. As a human being I feel the same for Syria, Kashmir, Palestine.

Why are devs forced to #StandWithUkraine? why not #StandWithKashmir, #StandWithSyria, #StandWithPalestine ?

Introduce "Who do you want to stand with?" while installing composer maybe.

**Comment by mohssineAboutaj** [comment]
Unfortunately, we know that the West has double standards, and there is no official solution, but I found one
just put this code in your `.bashrc` or `.zshrc` or whatever **CLI** you are using, the code bellow will replace the word "Ukraine" with "Palestine"
```sh
alias composer="2> >(sed 's/ukrain/palestine/g') composer --ansi"
```
if you just wanna remove the support put this line:
```sh
alias composer="2> >(grep -av 'StandWith') composer --ansi"
```

i got the idea from [this answer](https://github.com/composer/composer/issues/10807#issuecomment-1696563937) go and check it
